jobs/config.json

```diff
@@ -1129,7 +1129,7 @@
       "--gcp-node-image=gci",
       "--gcp-zone=us-central1-f",
       "--provider=gce",
-      "--test_args=--ginkgo.focus=\\[Feature:(Audit|BlockVolume|PodPreset|PVCProtection|ExpandPersistentVolumes)\\]|Networking --ginkgo.skip=Networking-Performance|IPv6 --minStartupPods=8",
```

Comment:
:-\ gah

Reply:
we can fix in a follow up, xref https://github.com/kubernetes/test-infra/pull/7070

It's hard for people to get it right with so many config files to look at 🙃